draft-ecahc-moderation.md

```diff
@@ -191,21 +191,34 @@ IETF's various participation channels: a moderator team for the IETF.
 The creation of this team intends to address the issues identified
 in {{motiv}}.
 
-## Scope
+## Composition
+
+The moderator team will initially consist of no less than five
+individuals.  The IETF Chair appoints and replaces members of the
+moderator team.  In selecting members, the IETF Chair will take into
+account geographic coverage, expected and unexpected absenses, and
+team diversity.  The moderation team may expand or contract
+based on operational experience.  Apart from appointing and replacing
+moderators, the IETF Chair SHALL refrain from the day-to-day operation
+and management of the moderator team. The moderator team MAY decide to
+consult with the IETF Chair when needed.
+
+Because the IESG and IAB are in the appeals chain for moderator team
+decisions (see {{appeals}}), the IETF Chair MUST NOT appoint a
+moderator who is serving on the IESG or IAB. Individuals serving on
+other bodies to which the NomCom appoints members, such as the IETF
+Trust or the LLC Board, as well as LLC staff and contractors SHALL
```

Comment:
I'm not sure about excluding the LLC staff/contractors.  Having some folks on their who part of their job role is to help with the moderation may be beneficial, although I can understand why you want to try and keep them separate.

Reply:
I think it's correct to exclude staff, because some (many?) moderation decisions may directly impact the standards process, and that is a strict NO for staff. However, as I've said before and will say again if necessary, staff must be able to perform emergency moderation when a posting is manifestly obscene, illegal or part of a detected DOS attack.

Reply:
@rgwilton as you know I initially thought Jay should do this job, but it's now raising some legal concerns if we break down the barrier.

Reply:
@rgwilton I'd like not to try to solve this in this PR.  If you feel strongly, could you open an issue?